tools/tsp-client/src/index.ts

```diff
@@ -218,6 +219,18 @@ const parser = yargs(hideBin(process.argv))
       await sortSwaggerCommand(argv);
     },
   )
+  .command(
+    "compare",
+    "Compare two Swaggers for functional equivalency. This is typically used to compare a source Swagger with a TypeSpec-generated Swagger to ensure that the TypeSpec project is functionally equivalent to the source Swagger.",
```

Comment:
Are we slightly contradicting ourselves here with what we're saying in the readme? Maybe we should reword to say: 
```suggestion
    "Compare two Swaggers for functional equivalency. This is typically used to compare a source Swagger with a TypeSpec project or TypeSpec generated Swagger to ensure that the TypeSpec project is functionally equivalent to the source Swagger.",
```